layerInfo.json

```
@@ -152,14 +141,17 @@
 				]
 			},
 			"ServiceClients": {
-				"dirs": ["packages/service-clients/"],
+				"dirs": ["packages/service-clients/", "@fluidframework/tinylicious-client"],
```

This isn't a dir :)

Shouldn't need to change this entry, the existing dir covers it.
```suggestion
				"dirs": ["packages/service-clients/"],
```